L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_5_cfi.py

```diff
@@ -120,14 +120,20 @@
 caloStage2Params.etSumEtaMin             = cms.vint32(1, 1, 1, 1, 1)
 caloStage2Params.etSumEtaMax             = cms.vint32(28,  26, 28,  28, 28)
 caloStage2Params.etSumEtThreshold        = cms.vdouble(0.,  30.,  0.,  30., 0.) # only 2nd (HT) and 4th (MHT) values applied
-caloStage2Params.etSumPUSType            = cms.string("LUT") # et threshold from this LUT supercedes et threshold in line above
-caloStage2Params.etSumBypassPUS           = cms.uint32(0)
+caloStage2Params.etSumMetPUSType         = cms.string("LUT") # et threshold from this LUT supercedes et threshold in line above
```

Comment:
@bundocka 
You are changing an already existing caloParams cfi file.  This is against our policy. 
If the changed line does the same thing, shouldn't this change also be made in all
other caloParams versions we have. 

We use etSumMetPUSType  starting from 2017_v1_4 and on.
Does it mean we cannot used versions earlier than v1_4 ?